.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,71 @@
+name: "unit tests"
+
+on: [ "pull_request", "push" ]
+
+jobs:
+    unit-tests:
+        name: "unit tests"
+
+        runs-on: "ubuntu-latest"
+
+        strategy:
+            matrix:
+                symfony-version:
+                    - "4.4.*"
+                    - "5.1.*"
+                php-version:
+                    - "7.2"
+                    - "7.3"
+                    - "7.4"
+                dependencies:
+                    - "lowest"
+                    - "highest"
+                include:
+                    - { symfony-version: "4.4.*", php-version: "8.0", dependencies: "highest" }
+                    - { symfony-version: "5.1.*", php-version: "8.0", dependencies: "highest" }
```

Comment:
8.0 fails with `lowest` dependencies ( simple-phpunit fails to be installed ) 🤔

Reply:
You should probably require a higher version of `simple-phpunit` (the latest for instance) in `composer.json`. It's OK because Simple PHPUnit is special, even the latest version supports very old versions of PHP and PHPUnit.

Reply:
test fail for 8.0 + lowest, leaving that to another PR.

https://github.com/azjezz/oauth2-server-bundle/actions/runs/636159892